exercises/luhn/description.md

```diff
@@ -7,8 +7,11 @@ The task is to write a function that checks if a given string is valid.
 
 Validating a Number
 ------
+Strings of length 1 or less are not valid. Spaces are allowed in the input,
+but they should be stripped before checking. All other non-digit characters
```

Comment:
I think the space stripping is already tested, so that a spaces only test would be superfluous.